src/server/assets/index.js

```diff
@@ -31,7 +32,8 @@ const getAssetsArray = () => {
       `Error parsing assets manifest. ${assetsManifestEnv} = ${process.env[assetsManifestEnv]}`,
     );
   }
-  return assets;
+
+  return assetsFilter(assets, service);
```

Comment:
Does this call invalidate the comment at https://github.com/bbc/simorgh/pull/2204/files#diff-eaae4b25e731b7caf088e06762d9ce1eR22?

Reply:
Soz, cant see what this is linking too, github is being 💩

Reply:
Sorry I meant the comment `Loops backwards as the client bundle is output first, but needs to be last.`